backend/core/database.py

```diff
@@ -775,9 +777,11 @@ def bootstrap_redis(self):
                     redis_pipeline.sadd("persistent-keys", entry[4])
                 redis_pipeline.execute()
 
+                # bootstrap BGP updates
                 query = (
-                    "SELECT DISTINCT key, timestamp FROM bgp_updates "
-                    "WHERE timestamp > NOW() - interval '1 hours'"
+                    "SELECT key, timestamp FROM bgp_updates "
+                    "WHERE timestamp > NOW() - interval '2 hours' "
+                    "ORDER BY timestamp ASC"
```

Comment:
I removed distinct which was I think the culprit, extended the interval to 2 hours as already stated for safety, and added the ordering so that even if a key appears multiple times with different timestamps, the most recent update prevails in redis. We can also deal with this with a more complex query to get only the most recent updates with the same key, but I opted for a faster and simpler query (the for-loop set will take care of the expiry time updates).